app.py

```diff
@@ -67,6 +67,14 @@ class Partnerships(Page):
     data = json.loads(pathlib.Path("_data/partnerships.json").read_text())
 
 
+@app.page
+class CodeOfConduct(Page):
+    Parser = MarkdownPageParser
+    content_path = "pages/coc.md"
+    template = "default.html"
+    slug = "coc"
+
+
```

Comment:
```suggestion
```

We don't need to build the coc again because it's built in the pages collection.